src/test/resources/application.conf.template

```diff
@@ -1,3 +1,6 @@
+slack-scala-client.http.useproxy = false
```

Comment:
I am not sure. Does this syntax even work?
Or should it be:
```
slack-scala-client {
  http {
    useproxy = false
  }
}
```?

Reply:
I run the tests locally with and without proxy, it worked.